connector-common/src/main/java/com/redhat/cloud/notifications/connector/EngineToConnectorRouteBuilder.java

```diff
@@ -72,6 +80,12 @@ public void configure() throws Exception {
                 .to(log(getClass().getName()).level("DEBUG").showHeaders(true).showBody(true))
                 .filter(incomingCloudEventFilter)
                 .process(this.incomingKafkaReinjectionHeadersProcessor)
+                .choice()
+                    .when(header(Constants.X_RH_NOTIFICATIONS_CONNECTOR_PAYLOAD_HEADER).isNotNull())
+                        .process(this.payloadDetailsRequestPreparer)
+                        .to(http(this.connectorConfig.getNotificationsEngineHostname()))
```

Comment:
Are you sure? The hostname from the ClowdApp configuration is not preceded with the protocol to use with it, and if I simply specify something like `localhost:1234` the build fails.

Reply:
Are you sure? because I checked log of email connector, and the recipients-resolver url contains the protocol.
I think we should give it a try without `http(...)` then add it if needed. WDYT?

Reply:
All right, I'll change it. I checked in ephemeral and I don't see a `url` key in the `cdappconfig.json` file, but